tests/integration/targets/aci_l3out_static_routes/tasks/main.yml

```diff
@@ -115,15 +115,36 @@
     prefix: 10.1.0.1/24
     state: query
   delegate_to: localhost
-  register: query_result
+  register: query_specific_mo
+
+- name: Verify query_specific_mo
+  assert:
+    that:
+    - query_specific_mo is not changed
+    - query_specific_mo.current|length == 1
+    - query_specific_mo.current[0].ipRouteP.attributes.rtCtrl == "bfd"
```

Comment:
This bfd line should only checked against ACI version >= 4.2 as on line 105 we only set the bfd flag for those APICs.

You can either remove that line or move it into another assert block with the same condition as in line 105.